src/main/java/com/hbm/render/block/ct/RenderBlocksCT.java

```diff
@@ -315,32 +312,32 @@ public void renderBlockAsItem(Block block, int meta, float mult) {
 		GL11.glRotatef(90.0F, 0.0F, 1.0F, 0.0F);
 		GL11.glTranslatef(-0.5F, -0.5F, -0.5F);
 		
-		tess.startDrawingQuads();
-		tess.setNormal(0.0F, -1.0F, 0.0F);
+		Tessellator.instance.startDrawingQuads();
```

Comment:
Small nit - it would be slightly faster to store the tessellator in a local var if accessing Tesselator.instance multile times; otherwise angelica is running through and replacing each of these with a .get() which might include a ThreadLocal lookup.